events/_posts/2015-04-10-kycc-wic.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Women in Computing Celebration
+start-time: 10:00am
+end-time: 6:30pm
+location: 38.061296, -84.499932
+location-title: BCTC
+---
+
+Grace Hopper Women in Computing Celebration. A regional conference hosted by the ACM Women in Computing chapter at BCTC.
```

Comment:
I think most people locally will know what BCTC is without spelling it out. Initials make it look clean.